### DISCUSSION TITLE
Need help for blinking cursor and folder background color.

### DISCUSSION THREAD

**Discussion by Rhotbhar** [BryanJenksCommunity/FAQ#151] [Q&A] [answered]
Hi, im using obsidan since more than a year.

I use the free template for signing up to the newsletter.
I am happy to use it. 

Problem for me are the amount oft new options through the premade template.

Sometimes if i write some notes the cursor is blinking. If this is happening i cant write and if i press the enter key to go to the next row or regardless try to write, the blinking cursor is jumping to another letter. Give it a solution for this?

The premade Folders, like inbox, have a background color, can someone tell me how is it possible and how i can do it with selfmade folders? 

I hope someone can help me, i am a bit helpless.

Also i hope my english is understandable, i am not a native speaker.

Best regards,
Rhotbhar

**Comment by tallguyjenks** [comment, marked as answer]
All stylization and customization is done through the CSS file, you'd have to learn a little CSS code to start modifying things yourself, or get others to do it for you

**Reply by Rhotbhar** [reply]
Thank you for your kindness,
the color for the folders i have found and i was able to change it. 

But for the cursor i only found the color code.
But the color is fine. The only problem is that i get problems that it is jumping to other rows and letter, and after a lot of try out, it normal functions are restored till it changes once more.
Could it be a possibility that some plugin  is the cause of the problem?

**Reply by tallguyjenks** [reply]
probably vim, i forgot to remove it when i export the template vault >.>

**Reply by Rhotbhar** [reply]
Everythings fine, it was indeed VIM.
Now everythings working like a charm.
im happy! Thank you :D
